diagnostics/mixed_layer_depth/settings.jsonc

```diff
@@ -0,0 +1,38 @@
+// mixed_layer_depth MDTF diagnostic settings file
```

Comment:
Please update the settings file to the new format ([documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html), [example POD](https://github.com/NOAA-GFDL/MDTF-diagnostics/blob/feature/v3alpha3/diagnostics/example/settings.jsonc)) -- this will let you specify variable and coordinate names and units, etc. The corresponding framework update will be merged into the develop branch soon.